### DISCUSSION TITLE
Feature Multicursor

### DISCUSSION THREAD

**Discussion by phifa** [standardnotes/forum#2543] [Suggestions]
A multicursor functionality (like in Sublime, VS Code, etc.) would be so helpful and unique. It's extremily powerful for fast editing text. This could be a toggle to switch on/off for any editor extension.

Comments? Thoughts?

**Comment by atmoio** [comment]
Possibly something we could gain if we make a [Monaco](https://github.com/Microsoft/monaco-editor) based editor.

**Comment by phifa** [comment]
this sounds amazing. Can you elaborate? Are there plans? Would this be a separate editor or a feature for all/some editors?

**Comment by atmoio** [comment]
No definite plans, just brainstorming. It would be a completely separate editor.

**Comment by phifa** [comment]
should we do some sort of feature voting?

**Comment by atmoio** [comment]
I'd refer to this regarding how we handle incoming feature discussions: https://standardnotes.org/longevity

**Comment by phifa** [comment]
@mobitar  I assume that would also include a bunch of other things, like moving lines with `alt+up/down`, etc. Cool!
